kvbc/include/categorization/block_merkle_category.h

```diff
@@ -72,7 +72,7 @@ class BlockMerkleCategory {
 
   // Delete the given block ID as a genesis one.
   // Precondition: The given block ID must be the genesis one.
-  void deleteGenesisBlock(BlockId, const BlockMerkleOutput&, storage::rocksdb::NativeWriteBatch&);
```

Comment:
Can we put a comment that these functions return the number of keys actually deleted from disk? Applies to all categories.

Reply:
Nit: Can use `std::size_t` instead.